src/components/containerGradient/DisplayTitle/DisplayTitle.tsx

```diff
@@ -0,0 +1,68 @@
+import cx from 'classnames';
+import styles from './DisplayTitle.module.scss';
+import { ColorLamp } from '../types';
+
+export type Props = {
+  title: string;
+
+  children?: React.ReactNode;
+  // temp prop
+  animationState?: string;
+
+  // to remove padding
+  inDisplay?: boolean;
+
+  color?: ColorLamp;
+
+  image?: {
```

Comment:
why like this? just pass image if necessary

Reply:
changed